lib/cryptoexchange/exchanges/lykke/services/pairs.rb

```diff
@@ -0,0 +1,54 @@
+module Cryptoexchange::Exchanges
+  module Lykke
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Lykke::Market::API_URL}/AssetPairs/rate"
+        TARGETS = ['GBP', 'EUR', 'USD', 'CHF', 'ETH', 'BTC', 'JPY', 'LKK1Y', 'LKK2Y', 'LKK', 'ZAR', 'PLN', 'SEK', 'NOK', 'DILS', 'TRY',
```

Comment:
Hello @estherleongym seems like LYKKE api has endpoint that list all pairs they have in this url: 
https://public-api.lykke.com/api/AssetPairs/dictionary
I think if you use this and directly use `baseAssetId` and `quotingAssetId`
so we don't need to list target like this way

But I roughly checked there's some exception on output that
```
{
"id": "LKK2YCHF",
"name": "LKK2Y/CHF",
"accuracy": 5,
"invertedAccuracy": 5,
"baseAssetId": "91316bdc-f379-4442-98fd-f2c844995ca1",
"quotingAssetId": "CHF"
}
```
base id is wrong

```
{
"id": "GB032548USD",
"name": "GB032548",
"accuracy": 2,
"invertedAccuracy": 0,
"baseAssetId": "USD",
"quotingAssetId": "GB032548"
},
```
base id and quoting asset id seems swap, and market and some market required endpoint respond no content (maybe just don't put this in pair first)

and 

```
{
"id": "SHEELA",
"name": "SHEELA/EUR",
"accuracy": 2,
"invertedAccuracy": 6,
"baseAssetId": "SHEELA",
"quotingAssetId": "EUR"
},
```
you need to use id to call other market_pair required endpoint

Reply:
@Swingcloud @estherleongym 

Using this API seem to be a viable option.
I would just ignore all the bad pairs as stated above.

My thought process would be
1. Query https://public-api.lykke.com/api/AssetPairs/dictionary (set as the API url)
2. Adapt and assign base target directly

In the Ticker API
use the market pair base/target as the `id` under adapt
You would no longer need the strip functions and all, since the id will come from the market pair

If the marketpair is bad as @Swingcloud mentions, it will just be not found and ignored which is fine